model/connection_string.go

```diff
@@ -51,13 +53,13 @@ var (
 // The implementation is based on the official python SDK.
 // https://github.com/Azure/azure-iot-sdk-python
 type ConnectionString struct {
-	HostName        string `cs:"HostName" bson:"hostname"`
-	GatewayHostName string `cs:"GatewayHostName" bson:"gateway_hostname,omitempty"`
-	Name            string `cs:"SharedAccessKeyName" bson:"name,omitempty"`
-	DeviceID        string `cs:"DeviceId" bson:"device_id,omitempty"`
-	ModuleID        string `cs:"ModuleId" bson:"module_id,omitempty"`
-	Key             []byte `cs:"SharedAccessKey" bson:"access_key"`
-	Signature       string `cs:"SharedAccessSignature" bson:"-"`
+	HostName        string        `cs:"HostName" bson:"hostname"`
+	GatewayHostName string        `cs:"GatewayHostName" bson:"gateway_hostname,omitempty"`
```

Comment:
Should we also encrypt the hostname properties?

Reply:
I don't have a strong opinion; however, the hostname is not a secret by itself.
Unless you think we have to encrypt this field, too, I'd leave it as it is.

Reply:
No strong opinion from my side either, I was just questioning whether we should treat the host name as sensitive data.

Reply:
Ok, let's keep it non-encrypted then.